README.md

```diff
@@ -436,34 +457,42 @@ type ZeroFields struct {
 
 </details>
 
-#### Extra rules behind `-extra`
+### Extra rules behind `-extra`
 
 Adjacent parameters with the same type should be grouped together
 
-<details><summary><i>example</i></summary>
+<details><summary><i>Example</i></summary>
 
-```
+```go
 func Foo(bar string, baz string) {}
 ```
 
-```
+```go
 func Foo(bar, baz string) {}
 ```
 
 </details>
 
-### Installation
+## Installation
 
-`gofumpt` is a replacement for `gofmt`, so you can simply `go install` it as
-described at the top of this README and use it.
+`gofumpt` is a replacement for `gofmt`, so you can simply `go install` it:
+
+```sh
+go install mvdan.cc/gofumpt@latest
```

Comment:
you're moving the `go install` way down - IMHO this is a mistake as it is what most new users want quickly.